### PR TITLE
Docker/kubernetes host deployment (IDR-0.4.4)

### PR DESCRIPTION
**Pull request by manics**
Removes notebooks from the docker hosts- this is now handled inside the `imagedata/jupyter-docker` notebook image.

### CODE REVIEW

File: ansible/group_vars/dockermanager-hosts.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
-kubernetes_kube_version: 1.7.0
+kubernetes_kube_version: 1.7.4
 
-idr_jupyter_hub_image: "imagedata/jupyterhub-githubauth:0.4.0"
-idr_jupyter_notebook_image: "imagedata/jupyter-docker:develop"
-idr_jupyter_notebook_repo: https://github.com/IDR/idr-notebooks.git
-idr_jupyter_notebook_repo_version: "0.5.0"
+idr_prepull_docker_images:
+- "imagedata/jupyterhub-githubauth:latest"
+- "imagedata/jupyter-docker:develop"
```
Comment (joshmoore) [inline ansible/group_vars/dockermanager-hosts.yml]: Time to tag then? We could call the current state `0.5.0` and then when the remaining 2-3 PRs are merged, `0.6.0`.

Comment (manics) [reply ansible/group_vars/dockermanager-hosts.yml]: Either that, or just go ahead and tag the version we'll actually use?

Comment (sbesson) [reply ansible/group_vars/dockermanager-hosts.yml]: `imagedata/jupyter-docker:0.5.0` is getting built as we speak. We can probably update this once the build is completed.

**Comment by manics** [comment]
Updated

**Comment by sbesson** [comment]
:+1: @joshmoore any final addition?

**Comment by joshmoore** [comment]
Not from my side.

**Pull request merged**